tutorials/btp-hana-tenantdb-xsapp-create/btp-hana-tenantdb-xsapp-create.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Create an SAP HANA Tenant Database and an SAP HANA XS Classic Application
+description: Create and bind an SAP HANA tenant database to an SAP HANA XS classic application.
+auto_validation: true
+time: 15
+tags: [ tutorial>beginner, topic>Cloud, products>sap-business-technology-platform, tutorial>license]
+primary_tag: products>sap-hana-service-for-sap-btp
+author_name: Sylvia Hördt
+author_profile: https://github.com/sapsy
+---
+
+## Prerequisites
+ - You have a subaccount in the SAP BTP, Neo environment.
+ - You're assigned the Administrator role for the subaccount.
+ - You've installed an SAP HANA tenant database system (MDC). See [Install Database Systems](https://help.sap.com/viewer/d4790b2de2f4429db6f3dff54e4d7b3a/Cloud/en-US/1261e6b87e174c05b774ea38fa3c8c51.html).
+
+## Details
+### You will learn
+How to create and bind an SAP HANA tenant database to an SAP HANA XS classic application using the SAP BTP cockpit, the SAP HANA cockpit, and the SAP HANA Web-Based Development Workbench
+
+To perform the different steps in this tutorial, you use different tools. You first create a tenant database, and then create the XS classic application on the database.
+
+![Overview](intro-flow.png)
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Create a database)]
+
+In your subaccount in the SAP BTP cockpit, you create a database on an SAP HANA tenant database system.
+
+1.  In the SAP BTP cockpit, navigate to a subaccount. For more information, see [Navigate in the Cockpit](https://help.sap.com/viewer/ea72206b834e4ace9cd834feed6c0e09/Cloud/en-US/fdeff7e68f64496eb8a1fb31f6a08b73.html).
+
+2.  Choose **SAP HANA / SAP ASE** > **Databases & Schemas** in the navigation area.
+
+    All databases available in the selected account are listed with their ID, type, version, and related database system.
+
+    **Tip**
```

Comment:
This should be part of the note.

Reply:
I cannot manage to have it inside the note, followed by a lign break. 
![Tip](https://user-images.githubusercontent.com/18330781/149509979-d15a57a7-499c-47e1-82db-663201e2762f.JPG)